Kubernetes/windows/debug/dumpVfpPolicies.ps1

```diff
@@ -37,6 +37,7 @@ foreach ($port in $ports) {
 	& $vfpCtrlExe /list-rule  /port $portGuid | Out-File $outfile -Encoding ascii -Append
```

Comment:
can you replace the /list-rule command with /get-rule-counter instead of appending it?
/get-rule-counter = /list-rule with rule counters